matrixbot/matrix.py

```diff
@@ -656,8 +656,8 @@ def _set_room_aliases(self, room_id):
             aliases = []
             room_dict_state = self.client.api.get_room_state(room_id)
             for e in room_dict_state:
-                if e['type'] == 'm.room.aliases':
```

Comment:
Just for the record. Could add a comment from where this change came in the matrix API. It could be relevant to check similar issues in the future.

Reply:
Other than that. LGTM.

:+1:  Thanks for your contribution.

Reply:
It looks coming from Synapse 1.10 release as commented in 1.11.0 release:

https://matrix.org/blog/2020/02/24/synapse-1-11-0-released